### ISSUE TITLE
Python files should use `snake_case`

### ISSUE THREAD

**Issue by dkmiller** [github/copilot-codespaces-demo#12]
[Python conventions](https://softwareengineering.stackexchange.com/a/308976) are to name modules (files) like `snake_case.py`:

> modules (filenames) should have short, all-lowercase names, and they can contain underscores

Some files in this repo are named using PascalCase instead, as if they were written in C\#:

https://github.com/
ConvertCommentsToCode.py

**Comment by joshaber** [comment]
You're absolutely right! Would you be interested in opening a PR making those updates?
